README.md

```diff
@@ -76,44 +82,51 @@ flake8 .
 ```
 
 ### Windows 10 Users
- Please install and set up the following packages first, upgrade if you find the package is already installed:
+ Please install and set up the following packages first. Upgrade if you find the package is already installed:
 
-* Download [Python3](https://www.python.org/downloads/). It is advisable to install the python package as an administrator. Click on the 'Add path' checkbox before moving on to the next step of the installation process. Run the 'python3 -V' command to see the version you have installed.
+* Download [Python3](https://www.python.org/downloads/). It is advisable to install the python package as an administrator. Click on the 'Add path' checkbox before moving on to the next step of the installation process. Run this command in your terminal to see the version you have installed.
+  ```sh
+  python3 -V
+  ```
 
 * Download [pip](https://pip.pypa.io/en/latest/installing/) and follow the instructions in the link as an installation guide.
 
+* Download and install [Git](https://git-scm.com/download/win). Clone the project to a fresh directory/folder from the `staging` branch. 
```

Comment:
Same comment as the one left above. If they clone, they get all the branches and can checkout to the available ones to see what's being done there. No need to explicitly add this.